Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM codait/max-base:v1.1.0
+FROM codait/max-base:v1.1.1
 
 # Fill in these with a link to the bucket containing the model and the model file name
 ARG model_bucket=http://max-assets.s3-api.us-geo.objectstorage.softlayer.net/max-named-entity-tagger
 ARG model_file=assets.tar.gz
 
-RUN wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file}
-RUN tar -x -C assets/ -f assets/${model_file} -v
-RUN rm assets/${model_file}
+WORKDIR /workspace
```

Comment:
Why do we need this? the base image sets the `WORKDIR`

Reply:
I looked up Docker document, which does not say anything about WORKDIR inheritance.  Are you aware of an official statement about what are inherited? I would agree with you if there is such a statement in the document, otherwise I'm leaning toward adding this (same as what the skeleton does).

Reply:
It does inherit - even if it is not documented it seems to work. Almost all other MAX models don't use the `WORKDIR` command so I think it's better to be consistent. 

We may want to remove that actually from the skeleton since it is confusing.

Reply:
@MLnick Yes it does inherit in the current docker version, but the issue is that this is undocumented. I'm not sure how our team wants to handle undocumented features in general -- they may subject to change between versions.

Reply:
For now I would prefer to be consistent with other models

Reply:
Most other models have `WORKDIR` set, right? When I did all the PRs I only spotted a few of them has no `WORKDIR`.